src/main/java/frc/robot/commands/ElevatorCalibrationAbsoluteEncoder.java

```diff
@@ -0,0 +1,31 @@
+// package frc.robot.commands;
+// import edu.wpi.first.wpilibj2.command.CommandBase;
+// import frc.robot.constants.ElevatorConstants;
+// import frc.robot.subsystems.Elevator;
+
+// public class ElevatorCalibrationAbsoluteEncoder extends CommandBase {
```

Comment:
If you're not using the class, feel free to delete it

Reply:
Hi Anthony, I might use it a little later, so I'll keep it for now